tensorly/decomposition/robust_decomposition.py

```diff
@@ -110,7 +110,7 @@ def robust_pca(X, mask=None, tol=10e-7, reg_E=1.0, reg_J=1.0,
 
         # Evolution of the reconstruction errors
         rec_X.append(T.norm(X - D - E, 2))
-        rec_D.append(np.max([T.norm(low_rank - D, 2) for low_rank in J]))
+        rec_D.append(T.max(T.tensor([T.norm(low_rank - D, 2) for low_rank in J])))
```

Comment:
If we remove `T.tensor` we get the following error `TypeError: max(): argument 'input' (position 1) must be Tensor, not list`
for PyTorch backend. To avoid copying we can use `torch.as_tensor` in PyTorch but I am not sure if it will work for other backends as I am not familiar with others

Reply:
Good point, I'll merge this and we can revisit later!